packages/web-components/fast-foundation/tsdoc.json

```diff
@@ -1,9 +1,48 @@
 {
-    "$schema": "https://developer.microsoft.com/json-schemas/tsdoc/v0/tsdoc.schema.json",
-    "tagDefinitions": [
-      {
-        "tagName": "@test",
-        "syntaxKind": "block"
-      }
-    ]
-  }
\ No newline at end of file
+  "$schema": "https://developer.microsoft.com/json-schemas/tsdoc/v0/tsdoc.schema.json",
+  "extends": ["@microsoft/api-extractor/extends/tsdoc-base.json"],
+  "tagDefinitions": [
+    {
+      "tagName": "@test",
```

Comment:
Is the test tag still needed?

Reply:
Probably not. It was already there so I just left it. I'll remove and give it a "test".

Reply:
The "@test" tag does not seem to be used anywhere and removing does not cause any errors or warnings, so I have removed it. I'm pretty sure it was copy/pasted from the jsdoc example web page anyway.

Reply:
Awesome, thanks!